File: core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/ui/menu/YouTubePlayerMenu.java
package com.pierfrancescosoffritti.androidyoutubeplayer.ui.menu;

import android.view.View;

public interface YouTubePlayerMenu {
    void show(View anchorView);
    void dismiss();

    void addItem(MenuItem menuItem);
    void removeItem(int itemIndex);
    int getItemCount();
}

File: core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/ui/menu/YouTubePlayerMenu.kt
package com.pierfrancescosoffritti.androidyoutubeplayer.core.ui.menu

import android.view.View

interface YouTubePlayerMenu {
    val itemCount: Int
    fun show(anchorView: View)
    fun dismiss()

    fun addItem(menuItem: MenuItem): YouTubePlayerMenu
    fun removeItem(itemIndex: Int): YouTubePlayerMenu
    fun removeItem(menuItem: MenuItem): YouTubePlayerMenu
}
